LinuxEX1.txt

```diff
@@ -0,0 +1,197 @@
+#firts exercise
+#a
+
+touch nameFile.txt
+cd nameFile.txt
+echo "antonna" > nameFile.txt
+cat nameFile.txt
+
+#b
+mkdir new
+
+#c
+cp ~/nameFile.txt ~/new
+cat new/nameFile.txt
```

Comment:
use cp to copy the file to the new directory instead of using cat. The cat command is used to display the contents of a file, not to copy it.